### ISSUE TITLE
Create CHS-jsPsych monorepo

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1327]
# TL;DR

Create [monorepo](https://www.atlassian.com/git/tutorials/monorepos#:~:text=What%20is%20a%20monorepo%3F,e.g%20via%20dependency%20management%20tools) for CHS-jsPsych NPM code/packages: any custom (CHS-specific) jsPsych plugins, and possibly other things like the `lookitJsPsychInit` function.

# Narrative

We need somewhere to store our CHS-specific jsPsych code/packages. 

# Implementation notes

A sensible approach is to start by recreating the existing jsPsych directory structure and build system for CI/CD etc. Doing it this way means we won't need to reinvent the wheel. It should also make it easier to contribute our plugins etc to jsPsych. However, we can definitely move away from jsPsych's patterns and tools if we have reasons to do so.

Here's some jsPsych info/docs that might be helpful:
https://www.jspsych.org/7.3/developers/configuration/
https://www.jspsych.org/7.3/developers/plugin-development/
https://www.jspsych.org/7.3/developers/extension-development/

# Acceptance Criteria

- [ ] CHS-jsPsych repo created under lookit project
- [ ] Set up build chain and NPM publishing
- [ ] Set up testing tools
- [ ] Set up GH triggers (if necessary?)

**Comment by okaycj** [comment]
This has been completed.  The monorepo can be found here -> https://github.com/lookit/lookit-jspsych.
